test/pkg_autocorrects_test.cc

```diff
@@ -19,95 +19,117 @@ using namespace std;
 auto logger = spdlog::stderr_color_mt("pkg-autocorrects-test");
```

Comment:
(commenting on this file as a whole)

I think it would be nice if these test cases were written as regular CLI or `*.autocorrects.exp` files so that we're using the regular Sorbet pipeline instead of a custom pipeline. However, it would be a lot more work to write them as those, since you need to create a new folder and ruby file for each package, plus a ruby file for the source where we want to trigger the autocorrect from. It’s also easier to read the test cases in `pkg_autocorrects_test.cc` since all the package files are right there to look at in one place versus spread out over a bunch of folders and files.